### DISCUSSION TITLE
multithreading: about trunk trun_thread fd_close

### DISCUSSION THREAD

**Discussion by jobo-zt** [baresip/restund#7] [Ideas]
modules\turn\alloc.c function static void destructor(void *arg)

Do you have time to complete it. It may be related to this
/* @TODO check fd deref cleanup on turn worker thread */
https://github.com/baresip/re/issues/1183

**Comment by sreimers** [comment]
It will take some time, the mutlithreading PR is a proof of concept and not production ready, yet.
In my setups the single core is enough to handle production setups with hundred of turn connections.
How many turn connections you have to handle?

**Comment by jobo-zt** [comment]
I'm just testing it. about trunk trun_thread
I am looking forward to it. I will submit a PR
I am working it on Windows and rpi
